website/docs/r/neptune_cluster_parameter_group.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "aws"
+page_title: "AWS: aws_neptune_cluster_parameter_group"
+sidebar_current: "docs-aws-resource-aws-neptune-cluster-parameter-group"
```

Comment:
Documentation page is missing a sidebar link in `website/aws.erb`

Reply:
done, also I found duplicate entry for neptune parameter group in this file and removed it.